docs/source/index.rst

```diff
@@ -14,7 +14,7 @@ Architecture
 
 .. sidebar:: Basilisk Info
 
-    .. image:: _images/static/Basilisk-Logo.svg
+    .. image:: _images/static/Basilisk-Logo.png
```

Comment:
This change seems unrelated to the PR. Might have slipped through?

Reply:
It is, it is a bug I saw while working on this branch.  I'll make a micro PR just for this fix and remove from this PR.

Reply:
That micro micro PR sounds like a little pain to do. I think it's also ok to let this small change through with this PR. I promise I won't tell.

Reply:
Ok, I'll leave it in.